docs/resources/machine_config_v2.md

```diff
@@ -312,6 +312,7 @@ The following attributes are exported:
 * `datastore_cluster` - (Optional) vSphere datastore cluster for virtual machine (string)
 * `disk_size` - (Optional) vSphere size of disk for docker VM (in MB). Default `20480` (string)
 * `folder` - (Optional) vSphere folder for the docker VM. This folder must already exist in the datacenter (string)
+* `graceful_shutdown_timeout` (Optional) Duration in seconds for wait for graceful shutdown of the VM. A force destroy will be performed when the value is zero (string)
```

Comment:
small nit, but maybe the wording `(Optional) Duration in seconds before the graceful shutdown of the VM times out and the VM is destroyed. A force destroy will be performed when the value is zero (string)` would sound better? This would also apply to other files which use this wording.

Reply:
fixed